api/login.js

```diff
@@ -13,6 +13,28 @@ var secured = function(req, res, next) {
   res.redirect("/");
 };
 
+router.get("/devlogin", function(req, res) {
+  //Block non-devs
+  if (req.hostname != "localhost") {
```

Comment:
can we also do a `process.env` check? not 100% sure but ppl might be able to spoof the request hostname

Reply:
yeah down to add